src/STAN.Client/STAN.Client.csproj

```diff
@@ -26,7 +26,7 @@
   <ItemGroup>
     <PackageReference Include="Google.Protobuf" Version="3.10.0" />
     <PackageReference Include="Google.Protobuf.Tools" Version="3.10.0" PrivateAssets="All" />
-    <PackageReference Include="NATS.Client" Version="0.9.0" />
+    <PackageReference Include="NATS.Client" Version="0.9.1-pre1" />
```

Comment:
@ColinSullivan1 Had to update to latest pre-rel of NATS as that one is signed. Or should not STAN.Client be signed?

Reply:
This is fine for now - we can get an official release out for core NATS, then release another version of STAN.